react/src/js/components/Consonant/Helpers/Helpers.js

```diff
@@ -413,8 +447,10 @@ export const getDateDescSort = cards => getDateAscSort(cards).reverse();
 export const getEventSort = (cards = [], eventFilter) => {
     const transformedCards = cards.map(card => ({
         id: card.id,
-        startDate: card.contentArea.dateDetailText.startTime || card.footer[0].left[1].startTime || '',
-        endDate: card.contentArea.dateDetailText.endTime || card.footer[0].left[1].endTime || '',
+        startDate: safeGet(card, 'contentArea.dateDetailText.startTime', safeGet(card, 'footer[0].left[1].startTime', '')),
+        ...((card.footer && card.footer[0].left.length > 1) ?
+            { endDate: card.footer[0].left[1].endTime } :
+            { endDate: card.contentArea.dateDetailText.endTime }),
```

Comment:
Would recommend updating `endDate` to use `safeGet` to avoid bug/issue above with ternary/spread (with `[]` case)